lib/docsparser.js

```diff
@@ -16,6 +16,11 @@ DocsParser.prototype.init = function(viewSettings) {
 };
 
 DocsParser.prototype.is_existing_comment = function(line) {
+    // handle ES2015 generator shorhand for Object
+    // example: * funcname(var) {}
+    if (line.search(/^\s*\*\s?[\$\_A-Za-z][\$\_\w]+\s?\([\$\_A-Za-z][\$\_\w]+\)\s?\{/) >= 0) {
+        return false;
+    }
     return ((line.search(/^\s*\*/) >=0) ? true : false);
```

Comment:
While you're on it could you please also replace this with `.test()` and remove the useless ternary operator